components/tooltip/tooltip.js

```diff
@@ -730,9 +730,13 @@ class Tooltip extends RtlMixin(LitElement) {
 			target = ownerRoot.querySelector(targetSelector);
 			target = target || ownerRoot?.host?.querySelector(targetSelector);
 		} else {
-			console.warn('<d2l-tooltip>: missing required attribute "for"');
 			const parentNode = this.parentNode;
 			target = parentNode.nodeType === Node.DOCUMENT_FRAGMENT_NODE ? ownerRoot.host : parentNode;
+
+			// reduce console pollution since Safari + VO prevents inadequate SR experience for tooltips during form validation when using 'for'
+			if (!(target?.localName === 'd2l-input-text' && target?.invalid)) {
```

Comment:
Note that we've used `tagName` (note it is upper-case whereas `localName` is lower-case) throughout the rest of our core code base. Just mentioning in case you wanted to be consistent, but either will work as we don't need to worry about a namespace.